tests/test_cli_core.py

```
@@ -0,0 +1,1392 @@
+        os.chdir(tmp_path)
+        task_path = tmp_path / "structured-task.md"
+
+        create_example_task(str(task_path))
+
+        content = task_path.read_text()
+        # Should contain common task sections
+        assert "#" in content  # Has headings
+        # Either from template or fallback, should have some content
+        assert len(content) > 50  # Reasonable length
+
+    def test_create_example_task_uses_template_if_available(self, tmp_path, capsys):
```

Ruff (0.14.14)

[warning] Unused method argument: `capsys`

(ARG002)